lib/system/nimscript.nim

```diff
@@ -256,39 +202,31 @@ proc exec*(command: string) {.
   raises: [OSError], tags: [ExecIOEffect, WriteIOEffect].} =
   ## Executes an external process. If the external process terminates with
   ## a non-zero exit code, an OSError exception is raised.
-  ##
-  ## **Note:** If you need a version of `exec` that returns the exit code
-  ## and text output of the command, you can use `system.gorgeEx
```

Comment:
It's also nothing that has to be changed as part of the PR, but in the future, `gorgeEx` and `writeFile` also need to be restricted, both a compile- and execution-time.